dvc/repo/__init__.py

```diff
@@ -353,15 +353,25 @@ def graph(self, stages=None, from_directory=None):
 
         return G, G_active
 
-    def pipelines(self, from_directory=None):
+    @staticmethod
```

Comment:
Or maybe
```
def pipelines(self, from_directory=None, active=False):
    import networkx as nx

    G = self.graph(from_directory=from_directory, active=active)

    return [
        G.subgraph(c).copy() for c in nx.weakly_connected_components(G)
    ]
```

and then use it down below as
```
for G in self.pipelines(from_directory, active=True):
```
?

Reply:
I believe you are right